nixos/modules/services/misc/ananicy.nix

```diff
@@ -116,7 +116,6 @@ in
       } // (if ((lib.getName cfg.package) == (lib.getName pkgs.ananicy-cpp)) then {
         # https://gitlab.com/ananicy-cpp/ananicy-cpp/-/blob/master/src/config.cpp#L12
         loglevel = mkOD "warn"; # default is info but its spammy
-        cgroup_realtime_workaround = mkOD config.systemd.enableUnifiedCgroupHierarchy;
```

Comment:
This should probably always be true now?

Reply:
The default is true: https://gitlab.com/ananicy-cpp/ananicy-cpp/-/blob/master/src/config.cpp#L22

Reply:
but added it for clarity now